pkg/security/secl/rules/ruleset.go

```diff
@@ -118,21 +119,24 @@ func (rd *RuleDefinition) MergeWith(rd2 *RuleDefinition) error {
 
 // ActionDefinition describes a rule action section
 type ActionDefinition struct {
-	Set *SetDefinition `yaml:"set"`
+	Set              *SetDefinition              `yaml:"set"`
+	RefreshUserCache *RefreshUserCacheDefinition `yaml:"refresh_user_cache"`
```

Comment:
Do you think that we could just use `callback` or `internal` ? I would prefer to not have to define specific things in the rulesset. If we can't, we could just use the RuleID to trigger the refresh function.